symfony/plugins/orangehrmAdminPlugin/client/jobCategory/components/dialogs/DeleteConfirmationDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
```

Comment:
better to move this into `symfony/client/components` since this may reuse most of the places

Reply:
done